src/FTPProfile.cpp

```diff
@@ -82,7 +81,6 @@ FTPProfile::FTPProfile(const TCHAR * name) :
 }
 
 FTPProfile::FTPProfile(const TCHAR * name, const FTPProfile* other) :
-	m_parent(other->m_parent),
 	m_port(other->m_port),

```

Comment:
why is this necessary?

Reply:
needed a real string copy, not only reference - readded it but with string duplication